app/src/main/java/com/infomaniak/mail/data/api/ApiRepository.kt

```diff
@@ -276,6 +276,17 @@ object ApiRepository : ApiRepositoryCore() {
         return callApi(ApiRoutes.flushFolder(mailboxUuid, folderId), POST)
     }
 
+    fun generateAiProposition(prompt: String): ApiResponse<AiResult> {
+        val message = mapOf(
+            "type" to "user",
+            "content" to prompt,
+        )
+
+        val body = mapOf("messages" to arrayOf(message))
+
+        return callApi(ApiRoutes.ai(), POST, body)
```

Comment:
This body val isn't necessary, we usually don't create it when you can oneline the call

Reply:
Actually it changes in the very next PR